js/net.ts

```diff
@@ -141,20 +143,20 @@ export interface Conn extends Reader, Writer, Closer {
  * `127.0.0.1:` or `[::1]:0`, a port number is automatically chosen. The
  * `addr()` method of `Listener` can be used to discover the chosen port.
  *
- * See `dial()` for a description of the network and address parameters.
+ * See `dial()` for a description of the address parameters and network options.
  */
-export function listen(network: Network, address: string): Listener {
-  const res = sendSync(dispatch.OP_LISTEN, { network, address });
-  return new ListenerImpl(res.rid, network, res.localAddr);
+export function listen(
+  address: string,
+  options: NetworkOptions = { network: "tcp" }
```

Comment:
@piscisaureus Is this the correct way to set optional arguments? I feel like this is brittle but I don't know the right pattern.

Reply:
It's correct

Reply:
Since there is only one option, this works. Generally I would use `{}` as the default value, and in the function body do something like:
`options = { network: "tcp", host: "localhost", foo: "defaultvalue", ...options };`